templates/CRM/Contribute/Form/AdditionalPayment.tpl

```diff
@@ -97,12 +97,12 @@
           </tr>
           <tr class="crm-payment-form-block-fee_amount">
             <td class="label">{$form.fee_amount.label}</td>
-            <td{$valueStyle}>{$form.fee_amount.html}</td>
```

Comment:
this looks like copy & paste to me `$valueStyle` is only otherwise used in Message Templates